common-tasks/working-with-office/en.md

```diff
@@ -0,0 +1,8 @@
++++
+title = "Working with office"
++++
+# Working with office
+
+Solus comes pre-installed with LibreOffice, an office suite that is capable of replacing Microsoft Office for the most common tasks.
+Unless you are a heavy power user, LibreOffice should provide you with all the tools needed to solve your daily office challenges.
+LibreOffice Writer users may want to install the package `libreoffice-common-dictionaries` from Software Center to automatically enable spell-checking support.
```

Comment:
There should be a newline before this, otherwise it isn't going to render correctly.

There should be a subsection (using `##`) called Dictionaries, in my opinion. Or Spell Checking. Also `libreoffice-voikko` should be mentioned for Finnish.